driver_test.go

```diff
@@ -297,7 +242,7 @@ func TestQueryContext(t *testing.T) {
 	}
 
 	// Drop table.
-	err = executeDdlApi(conn, []string{`DROP TABLE TestQueryContext`})
+	_, err = db.ExecContext(ctx, `DROP TABLE TestQueryContext`)
```

Comment:
This line looks short enough to merge with the line below 👍

Reply:
changed